src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const make = (x) => {
```

Comment:
```suggestion
  const adder = (x) => {
```
Не дуже зрозуміла назва make